### PR TITLE
Professional Development Content

### PR DESCRIPTION
**Pull request by ElizabethlawalCYF**
User stories and sprints

Added PD content on on agile methodology. 

Do you know how to add slides? I have pasted the link directly into the repo but I think there is a much smarter way of doing this.. 

Signed-off-by: ElizabethlawalCYF <101108952+ElizabethlawalCYF@users.noreply.github.com>

## What does this change?

Module:
Week(s):

### CODE REVIEW

File: docs/react/week-1/lesson.md [hunk 1]
```diff
@@ -507,6 +507,41 @@ In this exercise, we'll move the variables in `Logo`, `BestPokemon` and `CaughtP
 11. In the `BestPokemon.js` file replace the existing usage of `abilities` with the `abilities` **prop**. You should still see the Pokemon ability names in your web browser.
 12. **(STRETCH GOAL)** Repeat the process with the `date` variable in the `CaughtPokemon.js` file.
 
+##  Professional Development
+
+**Session: User stories and sprints**
```
Comment (SallyMcGrath) [inline docs/react/week-1/lesson.md:512]:
```suggestion
### Session: User stories and sprints
```

File: docs/react/week-1/lesson.md [hunk 2]
```diff
@@ -507,6 +507,41 @@ In this exercise, we'll move the variables in `Logo`, `BestPokemon` and `CaughtP
...
+
+React module will recap Agile methodology through classroom discussions and group projects.
+
+https://docs.google.com/presentation/u/0/?tgif=d
+
+
+**
```
Comment (SallyMcGrath) [inline docs/react/week-1/lesson.md:519]:
```suggestion
#### Client Brief
```

File: docs/react/week-1/lesson.md [hunk 3]
```diff
@@ -507,6 +507,41 @@ In this exercise, we'll move the variables in `Logo`, `BestPokemon` and `CaughtP
...
+Task**
```
Comment (SallyMcGrath) [inline docs/react/week-1/lesson.md:520]:
```suggestion

```

File: docs/react/week-1/lesson.md [hunk 4]
```diff
@@ -507,6 +507,41 @@ In this exercise, we'll move the variables in `Logo`, `BestPokemon` and `CaughtP
...
+**Session: User stories and sprints**
+
+React module will recap Agile methodology through classroom discussions and group projects.
+
+https://docs.google.com/presentation/u/0/?tgif=d
+
+
+**
+Task** 
+Client Brief:
```
Comment (SallyMcGrath) [inline docs/react/week-1/lesson.md:521]:
```suggestion
```

File: docs/react/week-1/lesson.md [hunk 5]
```diff
@@ -507,6 +507,41 @@ In this exercise, we'll move the variables in `Logo`, `BestPokemon` and `CaughtP
...
+
+A small hotel owner, after many years in business, has decided to go to the next level, perform renovations and then go online. An important part of this journey will be to build an online hotel booking app.
```
Comment (SallyMcGrath) [inline docs/react/week-1/lesson.md:523]:
```suggestion
A small hotel owner, after many years in business, has decided to perform renovations and then go online. An important part of this journey will be to build an online hotel booking app.
```

File: docs/react/week-1/lesson.md [hunk 6]
```diff
@@ -507,6 +507,41 @@ In this exercise, we'll move the variables in `Logo`, `BestPokemon` and `CaughtP
...
+
+The hotel owner has provided some information:
+
+The hotel owner would like to have a welcome page where people can find all the information they need about the hotel and from there, app users can go to the booking section.
```
Comment (SallyMcGrath) [inline docs/react/week-1/lesson.md:527]:
```suggestion
The hotel owner would like to have a welcome page where users can find all the information they need about the hotel. From there, users can go to the booking section.
```

File: docs/react/week-1/lesson.md [hunk 7]
```diff
@@ -507,6 +507,41 @@ In this exercise, we'll move the variables in `Logo`, `BestPokemon` and `CaughtP
...
+
+At the very beginning, they were planning to allow only hotel employees to use the app to complete bookings on behalf of customers over the phone.
+Since the hotel is currently under heavy renovation, there are no hotel or room pictures ready yet and they will only be ready at the very end of the project.
+
+Exercise (20 min)
```
Comment (SallyMcGrath) [inline docs/react/week-1/lesson.md:532]:
```suggestion
:::note Exercise (20 min)
```

This is how we demark an exercise block, :::note

File: docs/react/week-1/lesson.md [hunk 8]
```diff
@@ -507,6 +507,41 @@ In this exercise, we'll move the variables in `Logo`, `BestPokemon` and `CaughtP
...
+
+In small groups. Plan the app project and create user stories/scenarios.
+
+Resources required: Use Gitprojects to plan your project.
```
Comment (SallyMcGrath) [inline docs/react/week-1/lesson.md:536]:
```suggestion
Resources required: Use [GitHub Projects](https://github.com/features/issues) to plan your project. 
```

File: docs/react/week-1/lesson.md [hunk 9]
```diff
@@ -507,6 +507,41 @@ In this exercise, we'll move the variables in `Logo`, `BestPokemon` and `CaughtP
...
+The hotel owner has provided some information:
+
+The hotel owner would like to have a welcome page where people can find all the information they need about the hotel and from there, app users can go to the booking section.
+
+At the very beginning, they were planning to allow only hotel employees to use the app to complete bookings on behalf of customers over the phone.
+Since the hotel is currently under heavy renovation, there are no hotel or room pictures ready yet and they will only be ready at the very end of the project.
+
+Exercise (20 min)
+
+In small groups. Plan the app project and create user stories/scenarios.
```
Comment (SallyMcGrath) [inline docs/react/week-1/lesson.md:534]:
```suggestion
In small groups. Plan the app project and create user stories/scenarios on a Jam Board or sticky notes. Look at Fundamentals again to remind you.
```

File: docs/react/week-1/lesson.md [hunk 10]
```diff
@@ -507,6 +507,41 @@ In this exercise, we'll move the variables in `Logo`, `BestPokemon` and `CaughtP
...
+
+Resources required: Use Gitprojects to plan your project. 
+
+Exercise (40 min)
```
Comment (SallyMcGrath) [inline docs/react/week-1/lesson.md:538]:
```suggestion
:::note Exercise (40 min)
```

File: docs/react/week-1/lesson.md [hunk 11]
```diff
@@ -507,6 +507,41 @@ In this exercise, we'll move the variables in `Logo`, `BestPokemon` and `CaughtP
...
+
+In small groups. Plan the sprints, identify their objective and decide which stories should be included on each of them to ensure the customer can see meaningful demos of progress and be able to provide feedback.
+
```
Comment (SallyMcGrath) [inline docs/react/week-1/lesson.md:541]:
```suggestion
::: 
```

File: docs/react/week-1/lesson.md [hunk 12]
```diff
@@ -507,6 +507,41 @@ In this exercise, we'll move the variables in `Logo`, `BestPokemon` and `CaughtP
...
+Resources required: Gitprojects.
```
Comment (SallyMcGrath) [inline docs/react/week-1/lesson.md:542]:
```suggestion
Resources required: [GitHub Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/quickstart-for-projects).
```

They aren't called Gitprojects!

File: docs/react/week-1/lesson.md [hunk 13]
```diff
@@ -507,6 +507,41 @@ In this exercise, we'll move the variables in `Logo`, `BestPokemon` and `CaughtP
...
+
+
 :::
```
Comment (SallyMcGrath) [inline docs/react/week-1/lesson.md:545]:
```suggestion

```

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
Some minor notes and tweaks

**Comment by kfklein15** [comment]
@ElizabethlawalCYF , hope it is ok to close this PR. Since Esma is reviewing it all and moving it, this could generate a PR conflict ;) (your first one)

**Pull request closed without merging**